common/test/acceptance/tests/lms/test_progress_page.py

```diff
@@ -125,6 +125,7 @@ def _logged_in_session(self, staff=False):
 
 
 @attr(shard=9)
+@flaky(max_runs=20, min_passes=20)
```

Comment:
The tests failed, but only due to a timeout after passing these tests too much. I'm now confident in the fix, removing this commit